clean/hammond_pd_cprr.py

```diff
@@ -0,0 +1,149 @@
+from re import T
+import sys
+from numpy import result_type

```

Comment:
Doesn't look like you're using this import. We should never let VSCode putting garbage in our code.

Reply:
Got it.